main.go

```diff
@@ -35,20 +36,16 @@ type link struct {
 
 // D3 is the structure used to construct the data for the frontend D3 graph.
 type D3 struct {
-	Nodes           []node `json:"nodes"`
-	Links           []link `json:"links"`
-	RequestsUsed    int    `json:"requestsUsed"`
-	MaxRequests     int    `json:"maxRequests"`
-	LastUpdate      string `json:"lastUpdate"`
-	RefreshInterval int64  `json:"refreshInterval"`
+	Nodes      []node `json:"nodes"`
+	Links      []link `json:"links"`
+	LastUpdate string `json:"lastUpdate"`
```

Comment:
I'm planning for big upgrades for the D3 graph in the future - while storing the update time here seems to be easier now, in the future it might be possible that no extraneous data could be in what the graph sources.
Separating pure graph structure from other details is, in my view, the winning strategy for long-term maintenance of the project.